types/prototypes.d.ts

```diff
@@ -59,41 +59,25 @@ interface String {
    *                      (default: `false`)
    * @returns The cleaned slug string
    */
-  slugify: (options?: {
-    /**
-     * The replacement character to separate terms
-     * @defaultValue `'-'`
-     */
-    replacement?: string;
-
-    /**
-     * Replace all non-alphanumeric characters, or allow them?
-     * @defaultValue `false`
-     */
-    strict?: boolean;
-  }) => string;
+  slugify(options?: { replacement?: string; strict?: boolean }): string;
```

Comment:
For consistency's sake, can you change the options to `{ replacement, strict }?: [...]`, please?

Reply:
done